<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreathWidget</class>
 <widget class="QWidget" name="BreathWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frBreath">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="Oscilloscope" name="wgtBreathOscill" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblScale">
        <property name="text">
         <string>Масштаб</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbScale"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCalibrate">
        <property name="text">
         <string>Калибровка</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>460</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnRecord">
        <property name="toolTip">
         <string>Сигнал будет записан</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../kernel.qrc">
          <normaloff>:/images/SaveOK.png</normaloff>:/images/SaveOK.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Oscilloscope</class>
   <extends>QWidget</extends>
   <header location="global">oscilloscope.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbScale</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BreathWidget</receiver>
   <slot>on_scaleChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalibrate</sender>
   <signal>clicked()</signal>
   <receiver>BreathWidget</receiver>
   <slot>on_calibrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>872</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRecord</sender>
   <signal>clicked(bool)</signal>
   <receiver>BreathWidget</receiver>
   <slot>on_recChange(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_scaleChange(int)</slot>
  <slot>on_calibrate()</slot>
  <slot>on_recChange(bool)</slot>
 </slots>
</ui>
